worlds/aquaria/Options.py

```diff
@@ -168,6 +220,103 @@ class UnconfineHomeWater(Choice):
+    """
+    Make sure that there are no progression items at hard-to-reach or hard-to-find locations.
+    Those locations are very High locations (that need beast form, soup and skill to get),
+    every location in the bubble cave, locations where need you to cross a false wall without any indication,
+    the Arnassi race, bosses and minibosses. Useful for those that want a more casual run.
+    """
+    display_name = "No progression in hard or hidden locations"
+
+
+class NoProgressionSimonSays(Toggle):
+    """
+    Make sure that there are no progression items in the says area.
+    """
+    display_name = "No progression in Simon says area"
+
+
+class NoProgressionKelpForest(Toggle):
+    """
+    Make sure that there are no progression items in Kelp Forest (excluding Simon says area).
+    Can be useful to get smaller runs.
+    """
+    display_name = "No progression in Kelp Forest"
+
+
+class NoProgressionVeil(Toggle):
+    """
+    Make sure that there are no progression items in the Veil. Can be useful to get smaller runs.
+    """
+    display_name = "No progression in the Veil"
+
+
+class NoProgressionMithalas(Toggle):
+    """
+    Make sure that there are no progression items in the Mithalas (city, castle and cathedral).
+    Can be useful to get smaller runs.
+    """
+    display_name = "No progression in Mithalas"
+
+
+class NoProgressionEnergyTemple(Toggle):
+    """
+    Make sure that there are no progression items in the Energy Temple. Can be useful to get smaller runs.
+    """
+    display_name = "No progression in the Energy Temple"
+
+
+class NoProgressionArnassiRuins(Toggle):
+    """
+    Make sure that there are no progression items in the Arnassi Ruins. Can be useful to get smaller runs.
+    Note that if the Transportation turtle are not randomize, this include Simon Says area.
+    """
+    display_name = "No progression in Arnassi Ruins"
+
+
+class NoProgressionFrozenVeil(Toggle):
+    """
+    Make sure that there are no progression items in the Frozen Veil (including Ice Cavern and Bubble Cave).
+    Can be useful to get smaller runs.
+    """
+    display_name = "No progression in the Frozen Veil"
+
+
+class NoProgressionAbyss(Toggle):
+    """
+    Make sure that there are no progression items in the Abyss.
+    Can be useful to get smaller runs.
+    Has no effect if the objective is killing the four enemy gods.
+    """
+    display_name = "No progression in the Abyss"
+
+
+class NoProgressionSunkenCity(Toggle):
+    """
+    Make sure that there are no progression items in the Sunken City.
+    Can be useful to get smaller runs.
+    Has no effect if the objective is killing the four enemy gods.
+    """
+    display_name = "No progression in the Sunken City"
+
+
+class NoProgressionBody(Toggle):
+    """
+    Make sure that there are no progression items in the Body (including the before boss transturtle room.
+    and the boss location. Can be useful to get smaller runs.
```

Comment:
```suggestion
    """
    Make sure that there are no progression items in the Body (including the before-boss transturtle room
    and the boss location). Can be useful to get smaller runs.
```
The change to this one added a period instead of [tension relief](https://xkcd.com/859/). Also maybe a "-" is good here too.

Reply:
Oups. It is indeed better like that.